templates/web/pages/dashboard/display_orgs_table.tt.html

```diff
@@ -1,16 +1,20 @@
 <!-- start templates/[% template.name %] -->
 
-[% IF has_orgs %]
 <form method="GET" action="/cgi/display_org_table.pl">
     <label for="name">[% lang("name") %]:</label>
     <input type="text" id="name" name="name" value="[% name | html %]">
-    
+
     <label for="valid_org">[% lang("verified_status") %]:</label>
-    <input type="text" id="valid_org" name="valid_org" value="[% valid_org | html %]">
+    <select id="valid_org" name="valid_org">
+        <option value="unreviewed" [% IF valid_org == 'unreviewed' %]selected[% END %]>Unreviewed</option>
```

Comment:
We should add an option for the case of no selection:
```suggestion
        <option value="" [% IF !(valid_org) %]selected[% END %]></option>
        <option value="unreviewed" [% IF valid_org == 'unreviewed' %]selected[% END %]>Unreviewed</option>
```